src/overlays/actors/ovl_En_Changer/z_en_changer.c

```diff
@@ -216,31 +216,31 @@ void EnChanger_OpenChests(EnChanger* this, PlayState* play) {
     f32 zPos;
     f32 yPos;
     f32 xPos;
-    s16 temp_s0;
-    s16 temp_s0_2;
+    s16 openedChestSide;
+    s16 unopenedChestItemType;
     EnBox* left;
     EnBox* right;
 
     left = this->leftChest;
     right = this->rightChest;
 
     if (this->timer == 0) {
-        temp_s0_2 = temp_s0 = this->chestOpened; // Required to use the right registers
+        unopenedChestItemType = openedChestSide = this->chestOpened; // Required to use the right registers
 
-        switch (temp_s0_2) {
+        switch (openedChestSide) {
             case CHEST_LEFT:
                 xPos = right->dyna.actor.world.pos.x;
                 yPos = right->dyna.actor.world.pos.y;
                 zPos = right->dyna.actor.world.pos.z;
 
                 if (this->rightChestGetItemId == GI_DOOR_KEY) {
-                    Actor_Spawn(&play->actorCtx, play, ACTOR_EN_EX_ITEM, xPos, yPos, zPos, 0, 0, 0, 0xF);
+                    Actor_Spawn(&play->actorCtx, play, ACTOR_EN_EX_ITEM, xPos, yPos, zPos, 0, 0, 0, EXITEM_SMALL_KEY_CHEST);
                     Flags_SetSwitch(play, 0x32);
                 } else {
-                    temp_s0_2 = (s16)(this->rightChestGetItemId - GI_RUPEE_GREEN_LOSE) + EXITEM_CHEST;
+                    unopenedChestItemType = (s16)(this->rightChestGetItemId - GI_RUPEE_GREEN_LOSE) + EXITEM_GREEN_RUPEE_CHEST;
```

Comment:
why eliminate `EXITEM_CHEST` but keep `EXITEM_COUNTER`? They seem to serve the same purpose (to mark the start of an enum "subrange" instead of being a single enum value)

Reply:
`EXITEM_COUNTER` isn't used by this actor. I can take it out of `en_bom_bowl_man` if everyone's OK with the added scope. I wanted to keep this one focused.

Reply:
making changes in that actor now is fine, the PR is extremely small to begin with

Reply:
I'm fine any of keeping `EXITEM_CHEST`, removing only `EXITEM_CHEST`, or removing both, just curious why you think using enum value is better, because I think the original author (in #567) thought it was better to make a separate define. (Personally, my instinct is that the enum is better, but just trying to pay respects to Chesterton's Fence)

Reply:
The enum made more sense to me as well when I was referencing this function for rando. It makes it easier to group it with the other rewards.

I'll add the change for `EXITEM_COUNTER`.